src/io/flutter/preview/MoveHandle.java

```diff
@@ -14,7 +15,10 @@
 public class MoveHandle extends Handle {
   @Override
   public void paint(Graphics g) {
-    g.drawRect(0, 0, getWidth(), getHeight());
+    final Graphics2D g2 = (Graphics2D)g;
+    g2.setStroke(new BasicStroke(2));
+    g2.setColor(JBColor.BLUE);
```

Comment:
Is there a 'selection' color?

Reply:
Is this any better?
https://github.com/flutter/flutter-intellij/pull/1958